<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLCDNumber" name="lcdVorlauf">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdRuecklauf">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdWarmwasser">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdFrostschutz">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>141</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Vorlauf</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDial" name="QDialVorlauf"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>RÃ¼cklauf</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDial" name="QDialRuecklauf"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Warmwasser</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDial" name="QDialWarmwasser">
      <property name="minimum">
       <number>20</number>
      </property>
      <property name="maximum">
       <number>65</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Frostschutz</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDial" name="QDialFrostschutz"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>QDialVorlauf</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdVorlauf</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>QDialRuecklauf</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdRuecklauf</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>QDialWarmwasser</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdWarmwasser</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>QDialFrostschutz</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdFrostschutz</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
